scheduler/pkg/kafka/pipeline/v2.go

```diff
@@ -311,8 +311,7 @@ func updateResponseFromRawContents(res *v2_dataplane.ModelInferResponse) error {
 			output.Contents.Fp64Contents = make([]float64, getDataSize(output.Shape))
 			err = binary.Read(bytes.NewBuffer(rawOutput), binary.LittleEndian, &output.Contents.Fp64Contents)
 		case tyBytes:
-			output.Contents.BytesContents = make([][]byte, 1)
-			output.Contents.BytesContents[0] = rawOutput
+			output.Contents.BytesContents = convertRawBytesToByteContents(rawOutput)
```

Comment:
Is this called when returning results to user on request (REST) path?

Reply:
yes